src/utils.js

```diff
@@ -0,0 +1,53 @@
+import { getLayer } from 'ol-mapbox-style';
+import { Point } from 'ol/geom.js';
+import RenderFeature from 'ol/render/Feature.js';
+
+/**
+ * gets the ol layer for a given mapbox source-layer
+ * @param {string} mapboxLayerId mapbox source layer id
+ * @returns {import("ol/layer/Vector.js").default | import("ol/layer/VectorTile.js").default}
+ */
+export function getOlLayerForMbLayer(contentLayer, mapboxLayerId) {
```

Comment:
* JSDoc does not match function signature (1 vs 2 args)
* A better name for this function would be getLayerForMapboxSourceLayer, better describing what it does, and assuming we use `layer` and `source` for everything OpenLayers, and `Mapbox*` for everything Mapbox.
* The arguments should be named `contentLayer` and `mapboxSourceLayer`.